osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/Iec60870SimulatorApplication.java

```diff
@@ -0,0 +1,13 @@
+package org.opensmartgridplatform.simulator.protocol.iec60870;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Iec60870SimulatorApplication {
```

Comment:
Naming: Iec60870RtuSimulatorConfig vs. Iec60870SimulatorApplication. Should we rename it to Iec60870RtuSimulatorApplication?